designcompose/src/main/java/com/android/designcompose/Layout.kt

```diff
@@ -105,6 +107,11 @@ internal object LayoutManager {
         density = pixelDensity
     }
 
+    internal fun setGridNodeSize(nodeName: String, width: Int, height: Int) {
```

Comment:
Can we have collisions on this? I can imagine the same node name being in two docs (especially in the validation app!); I'm not sure it could happen in a single doc.

Reply:
Yes, there can be collisions. I thought about this but decided not to worry about it yet since I didn't want to put too much complexity into a solution that might not last. If we move forward I can add support for that or make a ticket for it.

Reply:
https://github.com/google/automotive-design-compose/issues/544